src/Prefix.jl

```diff
@@ -301,6 +301,8 @@ function setup(source::SetupSource{DirectorySource}, targetdir, verbose)
     if verbose
         @info "Copying content of $(basename(srcpath)) in $(basename(targetdir))..."
     end
+    # Create if missing. Will throw an IOError if targetdir is there, but not a directory.
+    isdir(targetdir) || mkdir(targetdir)
```

Comment:
This should be equivalent to
```suggestion
    mkpath(targetdir)
```